ui/app/store/actions.js

```diff
@@ -1450,11 +1450,12 @@ export function setPreviousProvider (type) {
 }
 
 export function updateAndSetCustomRpc (newRpc, chainId, ticker = 'ETH', nickname, rpcPrefs) {
+  const stringChainId = chainId?.toString()
```

Comment:
I'm not sure these three `toString()` calls are necessary - it looks like it was already a string in all cases, contrary to the JSDoc comments saying otherwise in the background.

Reply:
Well, in particular, the `chainId` in `frequentRpcListDetail` has always been a string, and the methods that set it (e.g. `updateAndSetCustomRpc`, `setCustomRpc`, etc.) all take string `chainId`s. The address book controller accepts only string `chainId`s. The network controller deals solely in string `chainId`s. They're all strings already.

Reply:
18569d2